adapter/validator/src/test/java/org/mobilitydata/gtfsvalidator/validator/GtfsTypeValidatorTest.java

```diff
@@ -204,26 +293,74 @@ void outOfRangeIntegerGenerateNotice() {
 
         Collection<Notice> result = underTest.validate(new ParsedEntity(
                 TEST_ID,
-                Map.of("integer_with_range", 67),
+                Map.of("out_of_range_latitude", -100.0f),
                 new RawFileInfo.RawFileInfoBuilder().filename(TEST_FILE_TST).build()
         ));
 
         assertEquals(1, result.size());
 
         Notice notice = new ArrayList<>(result).get(0);
-        assertThat(notice, instanceOf(IntegerFieldValueOutOfRangeNotice.class));
-        assertEquals("ERROR", ((IntegerFieldValueOutOfRangeNotice) notice).getLevel());
-        assertEquals(10, notice.getCode());
-        assertEquals("Out of range integer value", notice.getTitle());
+        assertThat(notice, instanceOf(FloatFieldValueOutOfRangeNotice.class));
+        assertEquals("ERROR", ((FloatFieldValueOutOfRangeNotice) notice).getLevel());
+        assertEquals(11, notice.getCode());
+        assertEquals("Out of range float value", notice.getTitle());
         assertEquals(TEST_FILE_TST, notice.getFilename());
-        assertEquals("Invalid value for field:`integer_with_range` of entity with id:`test_id` -- " +
-                        "min:-6 max:66 actual:67",
+        assertEquals("Invalid value for field:`out_of_range_latitude` of entity with id:`test_id` -- " +
+                        "min:-90.0 max:90.0 actual:-100.0",
                 notice.getDescription());
 
-        verify(mockIntegerValidator, times(1)).isInRange(
-                ArgumentMatchers.eq(Integer.valueOf(67)),
-                ArgumentMatchers.eq(-6),
-                ArgumentMatchers.eq(66)
+        verify(mockFloatValidator, times(1)).isInRange(
+                ArgumentMatchers.eq(Float.valueOf(-100f)),
+                ArgumentMatchers.eq(Float.valueOf(-90f)),
+                ArgumentMatchers.eq(Float.valueOf(90f)));
+    }
+
+    @Test
+    void outOfRangeLongitudeGenerateNotice() {
```

Comment:
Make sure we have corresponding in range tests that checks the range is inclusive (test for -90.0f et 90.0f)